components/Common/Card/index.module.css

```diff
@@ -0,0 +1,53 @@
+.container {
+  @apply max-w-full
+    bg-white
+    dark:bg-neutral-950;
+}
+
+.preview {
+  @apply mb-6
+    max-w-full
+    rounded p-4;
```

Comment:
More than one Tailwind class should not be on the same line 👀

Reply:
Thanks! Just pushed these changes